core/primitives/src/trie_key.rs

```diff
@@ -62,6 +62,7 @@ pub mod col {
     pub const BUFFERED_RECEIPT_GROUPS_QUEUE_DATA: u8 = 16;
     /// A single item of `ReceiptGroupsQueue`. Values are of type `ReceiptGroup`.
     pub const BUFFERED_RECEIPT_GROUPS_QUEUE_ITEM: u8 = 17;
+    pub const GLOBAL_CONTRACT_CODE: u8 = 18;
```

Comment:
nit: Can you add a comment including the value type (like above), please?

Reply:
f281709b1